configuration/init/systemd/tedge-mapper-c8y.service

```diff
@@ -5,8 +5,9 @@ After=syslog.target network.target mosquitto.service
 [Service]
 User=tedge-mapper
 ExecStart=/usr/bin/tedge_mapper c8y
-Restart=on-failure
+Restart=always
```

Comment:
Why change here to restart always but not the other services?
And anyway why do we want to restart always, this will cause the systemd to try restart even when it exits with code 0.

Reply:
Fixed